### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by Mughira12** [community/community#39195] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello can you please contact me at mughiraaa@gmail.com need to discuss something related to cardano tesnet

**Comment by david-engelmann** [comment, marked as answer]
@Mughira12 If you have a github specific question you can go through the [support dashboard](https://support.github.com/), or reach out at support@github.com. If your question is specific to the Cardano repository you will get better responses posting your question there, [select the relevant repo from this list](https://github.com/cardano-foundation).
